docs/run_on_aws.md

```diff
@@ -0,0 +1,55 @@
+# TorchServe on AWS
+
+The following guide will help you run TorchServe on AWS. You can install TorchServe on Ubuntu 16.04 or 18.04 EC2 instances. You can choose instances with GPUs (p or g-types are recommended) or only use CPUs (c or r-types are recommended). To test TorchServe, you can use a free-tier CPU-only instance type, like t2.medium.
+
+There a few ways you can use TorchServe on AWS:
+
+* The [AWS Deep Learning AMI (DLAMI)](https://docs.aws.amazon.com/dlami/latest/devguide/launch-config.html) on EC2
+* If you want fully managed inference, model monitoring, and autoscaling, use [Amazon SageMaker endpoints](https://aws.amazon.com/blogs/machine-learning/deploying-pytorch-models-for-inference-at-scale-using-torchserve/).
+* If you prefer Docker or Kubernetes, you can use [AWS Deep Learning Containers](https://docs.aws.amazon.com/deep-learning-containers/latest/devguide/setup.html).
+
+**Note:** Model serving and inference for very large models can be memory intensive, so choose your instance type with enough memory. If you use Amazon SageMaker endpoints, you can scale up or down, out or in, as needed.
+
+## Setup TorchServe on an AWS Deep Learning AMI
+
+1. Spin up the latest Deep Learning AMI (DLAMI). This should only take a few minutes.
+1. Log in to your DLAMI
+1. Install Java 11
+    ```bash
+    sudo apt-get install openjdk-11-jdk
+    ```
+1. Clone the [TorchServe repository](http://#)
+    ```bash
+    git clone https://github.com/pytorch/serve.git
+    cd serve
```

Comment:
I did this because the next steps you usually will do is play with the examples and for that you need the repo.
We try to avoid too many random buckets do to maintenance and security concerns. Happy to use someone's bucket if we have a plan for this - like the bucket used for the model zoo...
For now though, I think the yaml files are useful for any setup on any provider, and for local conda installs, so I think they belong in the repo.

Reply:
I'm not saying they don't belong in the repo - I think they're generally useful. I was just raising the question of whether `git clone` was the right way to distribute them, and you make a good point about the examples.